lib/vdsm/virt/qemuguestagent.py

```diff
@@ -781,3 +795,24 @@ def _qga_call_get_devices(self, vm):
         else:
             self.set_failure(vm.id)
             return {}
+
+    def _qga_call_get_vcpus(self, vm):
+        guest_info = {'guestCPUCount': -1}
```

Comment:
Do we want to return -1 or 0 when there are no online CPUs?

Reply:
Is that even possible? I mean, to have no on-line CPUs?

Reply:
I have tried that on Linux and it does not allow me to offline the last CPU.

Reply:
OK, so -1 means unexpected situation.